code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -41,6 +41,9 @@
 	var/pipe_state //icon_state as a pipe item
 	var/on = FALSE
 
+	/// Whether it can be painted
+	var/paintable = FALSE
```

Comment:
Yet another case of "this should be TRUE"

Reply:
Only (some) subtypes of `pipe` will have `paintable = TRUE`, so it makes sense to only make `pipe` `PAINTABLE = TRUE`. Also, here `paint` is a noop as you already have seen.

Reply:
ah gottcha